libs/ngx-aws-deploy/src/lib/deploy/index.ts

```diff
@@ -82,8 +90,10 @@ export default createBuilder(
       context.logger.info(`✔ Build Completed`);
     }
     if (buildResult.success) {
-      const filesPath = buildResult.outputPath as string;
-      const files = getFiles(filesPath);
+      const { outputPath } = await context.getTargetOptions(
+        targetFromTargetString(buildTarget.name)
+      );
```

Comment:
I needed this here since I'm using `@nrwl/web:webpack` for building my app and the outputPath property is not part of the buildResult object that is returned

Reply:
as you can see here @Jefiozie https://github.com/nrwl/nx/blob/master/packages/web/src/executors/webpack/webpack.impl.ts#L249